src/BizHawk.Client.EmuHawk/CustomControls/InputRoll/RollColumn.cs

```diff
@@ -28,11 +30,12 @@ public class RollColumn
 		/// </summary>
 		public bool Rotatable { get; set; }
 
-//		[JsonConstructor]
-		private RollColumn()
+		[JsonConstructor]
+		private RollColumn(string name, string text, ColumnType type)
```

Comment:
Uhh? Currently every property is de/serialised.

Reply:
The setters are private, so it's either this or adding `[JsonInclude]` to every property that has a private setter (while still keeping a dummy private constructor),